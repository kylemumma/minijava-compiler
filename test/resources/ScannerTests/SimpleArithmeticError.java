int a = b + c;[\~`\
System.out.println(a);
///////---------------------~~~..\\]\
/*
 * this is a multiline comment
 * 
 * 
 * 
 * 
 * 
 */