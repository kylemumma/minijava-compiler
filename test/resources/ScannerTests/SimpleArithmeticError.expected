INT ID(a) BECOMES ID(b) PLUS ID(c) SEMICOLON LBRACK <UNEXPECTED(\)> <UNEXPECTED(~)> <UNEXPECTED(`)> <UNEXPECTED(\)> PRINT LPAREN ID(a) RPAREN SEMICOLON